teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1923 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    ar_pcm_dataset
</pre><p>We usually only show a DataFrame's head (otherwise to much output on our website and more handy for <code>nbval</code> checks in our CI):</p><pre class="ql-syntax" spellcheck="false">ar_pcm_dataset.head()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
<p>Very good point. I implemented it notebook-wide.</p>